src/WinRT.Runtime/Interop/EventHandlerEventSource.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace WinRT.Interop
+{
+    /// <summary>
+    /// An <see cref="EventSource{TDelegate}"/> implementation for <see cref="EventHandler"/>.
+    /// </summary>
+    /// <remarks>This type is only meant to be used by generated projections.</remarks>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+#if EMBED
+    internal
+#else
+    public
+#endif  
+    sealed unsafe class EventHandlerEventSource : EventSource<EventHandler>
```

Comment:
Does this need to be public?  It looks like only the others are used from codewriters?

Reply:
Yeah I noticed the non-generic one wasn't actually used, I just figured it seemed consistent to expose this one too since it's kinda the same anyway? I'm happy to keep this internal though if you prefer. Wasn't sure if it could be useful at some point later.

Reply:
I saw this was actually a return type in ICommand, so probably needs to be public.